app/services/integrations/aggregator/credit_notes/create_service.rb

```diff
@@ -0,0 +1,154 @@
+# frozen_string_literal: true
+
+module Integrations
+  module Aggregator
+    module CreditNotes
+      class CreateService < BaseService
```

Comment:
Can't we inherit here from `Integrations::Aggregator::Invoices::BaseService` so that some methods can be reused?